app/models/meeting.rb

```diff
@@ -3,11 +3,23 @@ class Meeting < ApplicationRecord
 
   belongs_to :client, counter_cache: true
   belongs_to :advisor
-  
+
   before_save :add_upcoming_meeting_to_client
-  
+
+  scope :two_hours_from_now, lambda {
```

Comment:
An alternative approach might be to introduce a queuing system and queue up the alerts at the relevant times when the meeting is created

Reply:
Ah yes.. Could do.. Only thing to consider then is what if a meeting is rescheduled.. Would have to discard job and add another. Will have a think and see if there's a clean way to interact with the Que.

Reply:
We are simplifying to only one sms reminder the day before. I therefore think a cron task should do for now.